### PR TITLE
Update authors in posts

### PR DESCRIPTION
**Pull request by dalonsoa**
In two of the posts transfered form the old blog I appear as author, but I am not. It was like that in the original blog because there was no other choice, but now there is, so let's make sure the right people appears as author.

I'm not sure who attended RSECon22. I've added everyone I think was there, but to be honest I'm not sure who was actually there and participated, so amend as needed.

### CODE REVIEW

File: docs/posts/rsecon_2022.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 ---
 date: 2022-09-27
 authors:
-  - dalonsoa
+  - AdrianDAlessandro
+  - alexdewar
+  - cc-a
+  - dc2917
```
Comment (dc2917) [inline docs/posts/rsecon_2022.md:7]:
```suggestion
```

**Review by dc2917 — CHANGES_REQUESTED** [review]
Removed myself from RSECon22 post author list as I hadn't joined yet

**Review by jamesturner246 — APPROVED** [review]
LGTM :+1:

**Review by dc2917 — APPROVED** [review]

File: docs/posts/rsecon_2022.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 ---
 date: 2022-09-27
 authors:
-  - dalonsoa
+  - AdrianDAlessandro
```
Comment (AdrianDAlessandro) [inline docs/posts/rsecon_2022.md:4]: I don't remember being involved in this blog post. Does anyone know who wrote it?

Comment (dalonsoa) [reply docs/posts/rsecon_2022.md:4]: Possibly Dan Davies, now that I think about it. I should add him as co-author.

**Pull request merged**